app/src/main/res/values-de/strings.xml

```diff
@@ -191,7 +192,7 @@
     <string name="course_date_type_course_start">Kursstart</string>
     <string name="course_date_type_section_start">Sektionsstart</string>
     <string name="course_date_type_submission_publishing">Aufgabe</string>
-    <string name="course_date_type_submission_deadline">Abgabefrist</string> <!-- ToDo -->
```

Comment:
What about this change? What was the todo about?

Reply:
Finding a better translation

Reply:
@bengelhaupt I remember that this was one of your todos that I could remove. Is this correct?

Reply:
It was a todo annotation of me I forgot to remove. It was about translation if I remember correctly.